src/iterators.jl

```diff
@@ -193,12 +197,12 @@ end
 function reinit!(cache::InterfaceCache, face_a::FaceIndex, face_b::FaceIndex)
     reinit!(cache.a, face_a)
     reinit!(cache.b, face_b)
-    resize!(cache.dofs, length(cache.a.cc.dofs) + length(cache.b.cc.dofs))
-    for (i, d) in pairs(cache.a.cc.dofs)
+    resize!(cache.dofs, length(cache.a.dofs) + length(cache.b.dofs))
```

Comment:
Why not
```suggestion
    resize!(cache.dofs, length(celldofs(cache.a)) + length(celldofs(cache.b)))
```
?

Reply:
Perhaps this should be called `facedofs` (like `interfacedofs`, although that one has a slightly different meaning which warrants another name).